pkg/gatherers/conditional/gather_pod_definition.go

```diff
@@ -0,0 +1,91 @@
+package conditional
+
+import (
+	"context"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/insights-operator/pkg/gatherers"
+	"github.com/openshift/insights-operator/pkg/record"
+)
+
+// BuildGatherPodDefinition collects pod definition from pods that are firing one of the configured alerts.
+//
+// * Location in archive: conditional/namespaces/<namespace>/pods/<pod>.json
```

Comment:
Hmm so we already have e.g `conditional/namespaces/openshift-monitoring/pods/alertmanager-main-0/` dir in our sample archive. Wouldn't be better to move this json into this directory? IOW include directory with the pod name here.